docs/kubernetes_deploy.md

```diff
@@ -0,0 +1,162 @@
+<!--
+Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Deploying Model Analyzer on a Kubernetes cluster
+
+Model Analyzer provides support deployment on a Kubernetes enabled
+cluster using helm charts. You can find information about helm charts [here](https://helm.sh/).
+
+## Requirements
+
+* A [Kubernetes](https://kubernetes.io/) enabled cluster
+* [Helm](https://helm.sh/)
+
+## Using Kubernetes with GPUs
+
+1. **Install Kubernetes :** If you need to set up a Kubernetes cluster, you can follow the steps in the [NVIDIA Kubernetes Installation Docs](https://docs.nvidia.com/datacenter/cloud-native/kubernetes/install-k8s.html) to verify your installation, and troubleshoot any issues.
+2. **Set Default Container Runtime :** Using Kubernetes with GPU accelerated containers is tricky. Kubernetes does not yet support the `--gpus` options for running Docker containers, so you will need to register the `nvidia` runtime as the default for Docker on all GPU nodes. Follow the directions in the  [NVIDIA Container Toolkit Installation Docs](https://docs.nvidia.com/datacenter/cloud-native/kubernetes/dcgme2e.html#install-nvidia-container-toolkit-previously-nvidia-docker2).
+
+3. **Install NVIDIA Device Plugin :** Also required to use GPUs with Kubernetes is the NVIDIA Device Plugin, a daemonset that automatically enumerates the number of GPUs on your worker nodes, and allows pods to run on them. Follow the directions in the [NVIDIA Device Plugin Docs](https://docs.nvidia.com/datacenter/cloud-native/kubernetes/dcgme2e.html#install-nvidia-device-plugin) to deploy the device plugin on your cluster.
```

Comment:
"The NVIDIA Device Plugin is required to use GPUs with Kubernetes. The device plugin provides a daemonset that automatically .. to run on them."